src/anm/animation/element.js

```diff
@@ -2884,6 +2885,46 @@ Element.getMatrixOf = function(elm, m) {
     return t;
 };
 
+Element.getTranslate = function(elm) {
```

Comment:
By the way, you may do it `Element.prototype.getTranslate`, then it will be instance method, not static method, and element will be accessible as `this`.